components/dialog/index.css

```
@@ -66,23 +54,23 @@ governing permissions and limitations under the License.
 }
 
 .spectrum-Dialog--small {
-  inline-size: var(--spectrum-dialog-confirm-small-width);
+  inline-size: var(--mod-dialog-confirm-small-width, var(--spectrum-dialog-confirm-small-width));
 }
 
 .spectrum-Dialog--medium {
-  inline-size: var(--spectrum-dialog-confirm-medium-width);
+  inline-size: var(--mod-dialog-confirm-medium-width,var(--spectrum-dialog-confirm-medium-width));
 }
 
 .spectrum-Dialog--large {
-  inline-size: var(--spectrum-dialog-confirm-large-width);
+  inline-size: var(--mod-dialog-confirm-large-width,var(--spectrum-dialog-confirm-large-width));
 }
 
 .spectrum-Dialog-hero {
   grid-area: hero;
-  height: var(--spectrum-dialog-confirm-hero-height);
+  height: var(--mod-dialog-confirm-hero-height,var(--spectrum-dialog-confirm-hero-height));
 
-  border-top-left-radius: var(--spectrum-dialog-confirm-border-radius);
-  border-top-right-radius: var(--spectrum-dialog-confirm-border-radius);
+  border-top-left-radius: var(--mod-dialog-confirm-border-radius,var(--spectrum-dialog-confirm-border-radius));
+  border-top-right-radius: var(--mod-dialog-confirm-border-radius,var(--spectrum-dialog-confirm-border-radius));
```

```suggestion
  border-top-left-radius: var(--mod-dialog-confirm-border-radius, var(--spectrum-dialog-confirm-border-radius));
  border-top-right-radius: var(--mod-dialog-confirm-border-radius, var(--spectrum-dialog-confirm-border-radius));
```
Just a formatting consistency thing to improve readability---can you make sure there are spaces before var() and the comma throughout this file?

---

Could you make sure this change is made throughout the entire index.css file?